### PR TITLE
BlockingStorageTest (DEBUG)

### PR DESCRIPTION
**Pull request by Nimmaster**

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 I can review this

**Comment by 0crat** [comment]
This pull request [#66](https://github.com/artipie/asto/issues/66) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/755f5044-07c7-410a-8f7c-87355d5e6dd7); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by olenagerasimova** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/asto/issues/66#issuecomment-587479172))

@olenagerasimova The user @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) resigned from [#66](https://github.com/artipie/asto/issues/66), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/asto/issues/66#issuecomment-587479172))

@olenagerasimova Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by g4s8** [comment]
@0crat assign @paulodamaso

File: src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        final String content = "Hello world";
+        Files.writeString(
+            this.file,
+            content,
+            StandardOpenOption.WRITE,
+            StandardOpenOption.CREATE,
+            StandardOpenOption.TRUNCATE_EXISTING
+        );
+        final BlockingStorage asto = new BlockingStorage(new FileStorage(this.temp));
+        MatcherAssert.assertThat(
```
Comment (paulodamaso) [inline src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java]: @Nimmaster Please, add a description to the test case failure

**Comment by 0crat** [comment]
> @0crat assign @paulodamaso  ([here](https://github.com/artipie/asto/issues/66#issuecomment-587494453))

@g4s8 This pull request [#66](https://github.com/artipie/asto/issues/66) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c6ad7571-eaec-4c36-9837-7ef7bbfccd8c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
@0crat[/z](https://www.0crat.com/u/0crat) I see that you unassigned this issue; the order is still assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by Sammers21** [comment]
We already have a test, which do the same thing. Not sure if we need those changes.

**Comment by paulodamaso** [comment]
@Sammers21 Now I'm puzzled... What's the meaning of making a PR of a test that we already have?

**Comment by 0crat** [comment]
Job was finished in 18 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Pull request [#66](https://github.com/artipie/asto/issues/66) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request closed without merging**